index.html

```diff
@@ -153,6 +153,21 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
           };
         </pre>
         <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+        <p>
+          To <dfn data-export>create a CropTarget</dfn> with <var>element</var> as input, run the following steps:
+          <ol>
+            <li>
+              <p>Let <var>cropTarget</var> be a new object of type {{CropTarget}}.</p>
+            </li>
+            <li>
+              <p>Create <var>cropTarget</var>.<dfn data-dfn-for="CropTarget">[[\Element]]</dfn> initialized to <var>element</var>.</p>
+              <div class="note">
```

Comment:
On re-reading this after https://github.com/w3c/mediacapture-region/pull/24#issuecomment-1076252277 I see we need a serializable internal slot here. I propose:
 ```suggestion
             <li>
              <p>Let <var>handle</var> be a reference identifier to <var>element</var>.</p>.</p>
            </li>
              <p>Create <var>cropTarget</var>.<dfn data-dfn-for="CropTarget">[[\ElementHandle]]</dfn> initialized to <var>handle</var>.</p>
              <div class="note">
```
This is based on precedent in [generateCertificate](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-generatecertificate): _"Store the generatedKeyingMaterial in a secure module, and let handle be a reference identifier to it. ... Set certificate.[[[KeyingMaterialHandle]]](https://w3c.github.io/webrtc-pc/#dfn-keyingmaterialhandle) to handle."_

Reply:
It's plain text for an id.

> Does it make sense once the CropTarget is posted to another document?

The text _"in a manner that survives serialization"_ in the next sentence was meant to clarify this.

Reply:
weak reference is what I should have probably used from the start.